{"name": "test_update_post_with_invalid_id", "status": "skipped", "statusDetails": {"message": "XFAIL Fake API accepts string ID\n\nAssertionError: Expected 400 or 404 for invalid ID\nassert 500 in [400, 404]\n +  where 500 = <Response [500]>.status_code", "trace": "api = <api.posts_api.PostsAPI object at 0x000001EF8EC77A60>\n\n    @pytest.mark.negative_test\n    @pytest.mark.xfail(reason=\"Fake API accepts string ID\")\n    def test_update_post_with_invalid_id(api):\n        payload = negative_data[\"invalid_id_post\"]\n        res = api.update_post(\"abc\", payload)\n>       assert res.status_code in [400, 404], \"Expected 400 or 404 for invalid ID\"\nE       AssertionError: Expected 400 or 404 for invalid ID\nE       assert 500 in [400, 404]\nE        +  where 500 = <Response [500]>.status_code\n\ntests\\test_posts.py:82: AssertionError"}, "attachments": [{"name": "log", "source": "25fb15dc-513a-4b22-8881-f88a44f6ad5e-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "9985bbe0-337f-4b78-ae32-3085d9676934-attachment.txt", "type": "text/plain"}], "start": 1751300557252, "stop": 1751300558078, "uuid": "f137c45c-d66e-40e6-9268-f900279ff918", "historyId": "1c5e41e8af2790ca3aaa52307104bd68", "testCaseId": "1c5e41e8af2790ca3aaa52307104bd68", "fullName": "tests.test_posts#test_update_post_with_invalid_id", "labels": [{"name": "tag", "value": "@pytest.mark.xfail(reason='Fake API accepts string ID')"}, {"name": "tag", "value": "negative_test"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_posts"}, {"name": "host", "value": "DESKTOP-J0TB641"}, {"name": "thread", "value": "4944-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_posts"}]}